draft-ietf-tls-tls13.md

```diff
@@ -1567,8 +1567,8 @@ cipher_suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
   usual. Values are defined in {{cipher-suites}}. If the client is
-  attempting a PSK key establishment, it MUST only advertise the single
-  cipher suite associated with the PSK.
+  attempting a PSK key establishment, it MUST advertise at least one
+  cipher suite containing a Hash associated with the PSK.
```

Comment:
> Weren't you objecting in your thread that asymmetric MUSTs are confusing?

I did but ekr's justification seems fine that I changed my opinion on it :) If you think this is too much I could change it for a SHOULD. What do you think?

> 0-RTT has stricter requirements than resumption

I see. Thanks for the explanations!